.gitignore

```diff
@@ -27,3 +27,6 @@ wasm/*.wasm
 
 # app version string file
 /apps/version.rs
+
+# for annoying contributors using mac
+*.DS_Store 
```

Comment:
you can put this in your global git config e.g. https://gist.github.com/subfuzion/db7f57fff2fb6998a16c
```suggestion
```